cmd/admin-handlers-config-kv.go

```diff
@@ -201,6 +208,18 @@ func setConfigKV(ctx context.Context, objectAPI ObjectLayer, kvBytes []byte) (re
 		return
 	}
 
+	// Check if subnet proxy being set and if so set the same value to proxy of subnet
+	// target of logger webhook configuration
+	if result.SubSys == config.SubnetSubSys {
+		loggerWebhookSubnetProxy := result.Cfg[config.LoggerWebhookSubSys][config.SubnetSubSys].Get(logger.Proxy)
+		subnetProxy := result.Cfg[config.SubnetSubSys][config.Default].Get(logger.Proxy)
+		if loggerWebhookSubnetProxy != subnetProxy {
+			obj := result.Cfg[config.LoggerWebhookSubSys][subnet.LoggerWebhookName]
+			obj.Set(logger.Proxy, subnetProxy)
+			result.LoggerWebhookCfgUpdated = true
+		}
```

Comment:
After the proxy is set, the corresponding logger proxy also gets set.
However,  when the proxy is removed, the logger proxy is not getting removed.
This is because removing proxy invokes the `del-config-kv` api.
You may want to modify that api as well to handle this.

Reply:
Sure. Will do that